components/table/stories/table.stories.js

```diff
@@ -530,16 +634,30 @@ ThumbnailsCollapsible.args = {
 		},
 	],
 };
+ThumbnailsCollapsible.tags = ["!dev"];
+ThumbnailsCollapsible.storyName = "Thumbnails: collapsible";
 ThumbnailsCollapsible.parameters = {
-	chromatic: { disableSnapshot: true }
+	chromatic: { disableSnapshot: true },
 };
 
 /**
- * In addition to the overall table, individual rows can be designated as a drop target. Only one dropzone row should show at a time, but this example sets multiple at different parts of the table to test that they all display the same.
+ * The table body can accept dropped content.
  */
-export const RowDropzone = Template.bind({});
+export const BodyDropZone = Template.bind({});
+BodyDropZone.args = {
+	isDropTarget: true,
+};
+BodyDropZone.tags = ["!dev"];
+BodyDropZone.storyName = "Dropzone: body";
```

Comment:
Excellent, nice thought keeping those similar variants named similarly like this!